src/Player/Player.cpp

```diff
@@ -37,7 +40,145 @@ namespace Player {
+        vector<int> answer_cards;
+
+            if(this->hand->size() >= 5){
+                //Displaying warning message
+                Terminal::print(print_exchange_required);
+
+                //Displaying available cards to Player
+                for(int i=0; i<this->hand->size(); i++){
+                    print_hand.emplace_back("Card " + to_string(i) + " : Type = " + this->hand->get_cards().at(i)->getType_to_string() + ", Country: " + this->hand->get_cards().at(i)->getCountry() + ".");
+                }
+
+                //Prompt the user to choose three cards
+                answer_cards.emplace_back() = Terminal::print_select("Please input your desired [3] cards: "); /* Verify if user can actually input 3 values */
+
+                //Checking if cards are valid
+               /* if(this->hand->cardsValidForExchange(*//* IS VALID FOR EXCHANGE *//*)){
+                    exchange_desired = Terminal::print_select("Player [Color] has cards that are valid for exchange, would you like to proceed now?");
+                    if(exchange_desired){
+                        army_exchange += this->hand->exchange(*//*  INDICES *//* );
+                    }
+                }*/
+
+                //Exchanging cards
+                while(this->hand->size() >= 5){
+                    army_exchange += this->hand->exchange(&answer_cards[0]); /* ??????? check if conversion is correct */
+                }
+            }
+
+        //Therefore, the total armies available (A) to Player are
+        army_total = army_country + army_continent + army_exchange;
+
+        /*
+         * Then player places all armies on some of the countries it owns as they see fit.
+         */
+
+        //Looping around the number of armies available to place
+        do{
+            //Display available options (armies available + countries for positioning)
+            Terminal::print("You are now able to place your reinforcement armies. The total armies and countries available for positioning are the following: ");
+                vector<string> armies_print;
+                armies_print.emplace_back(army_total + " total armies available");
+                Terminal::print(armies_print);
+            /*INDEX = */
+
+                //Gathering countries owned by player and in map
+                    //vector<Countries *> countries_owned ~ already declared above
+                    //vector<Countries *> countries_in_map ~ already declared above
+                //Printing Utility
+                vector<string> options;
+                for(auto c : countries_owned) {
+                    options.emplace_back(c->get_name() + " (which has" + to_string(c->get_armies()) + " armies present).");
+                }
+                //Asking player to input data
+                options.emplace_back("\nPlease enter the country [#] to which you would like to place armies: ");
+                int answer_country = Terminal::print_select(options);
```

Comment:
The "\nPlease enter the country..." Should not be part of the options, but rather a separate Terminal::print() call.

Reply:
Makes sense, but how do I make the separate call appear between the country list and the cin>> from the print_select(options)?

Reply:
I would do 2 separate calls:
1. Terminal::print("Please enter the country......")
2. Terminal::print_select(options);

Reply:
Alrighty